src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+
+const Header = () => {
+  return (
+    <div className='header'>
```

Comment:
You've done well adding semantic tags throughout the app so far. This one should be <header>